packages/components/src/components/input-radio/shadow.tsx

```diff
@@ -112,9 +112,7 @@ export class KolInputRadio implements InputRadioAPI, FocusableElement {
 			state: this.state,
 			id: id,
 			ref: this.state._value === option.value ? this.catchRef : undefined,
-			class: clsx('kol-input-radio', {
-				['kol-input-radio--disabled']: Boolean(this.state._disabled || option.disabled),
-			}),
+			class: 'kol-input-radio kol-input-radio__input',
```

Comment:
Das entspricht leider nicht der BEM-Konvention, ein Tag kann nicht gleichzeitig Block und Element sein. 

```suggestion
			class: 'kol-input-radio__input',
```